src/bulma/pages/components/Order.vue

```diff
@@ -1,11 +1,14 @@
 <template>
     <div class="wrapper">
-        <a class="button quick-view-control is-pulled-right"
-            @click="quickView = true">
-            <span class="icon is-large">
-                <fa icon="ellipsis-h"/>
-            </span>
-        </a>
+        <div class="quick-view-control is-pulled-right">
```

Comment:
let's rename `quick-view-control` to `controls`, it's not dedicated to quick view anymore

let's rename the slot also to `controls` feels more accurate

Reply:
I renamed it to order-controls because controls had conflicts